__mocks__/react-relay.js

```diff
@@ -9,6 +9,6 @@ module.exports = {
   createRefetchContainer: component => component,
   graphql: Relay.graphql,
   commitMutation: (environment, { onCompleted, onError }) => {
-    onCompleted()
+    onCompleted && onCompleted()
```

Comment:
`onCompleted` is typed as optional, so this matches that behaviour and allows for mutations to fire-and-forget.